app/src/main/java/no/usn/mob3000/ui/components/info/ContentEditor.kt

```diff
@@ -43,22 +44,25 @@ fun ContentEditor(
             OutlinedTextField(
                 value = title,
                 onValueChange = onTitleChange,
-                label = { Text(stringResource(R.string.info_item_content_title)) },
+                label = { Text(stringResource(R.string.info_item_content_title), color = MaterialTheme.colorScheme.onBackground) },
+                textStyle = TextStyle(color = MaterialTheme.colorScheme.onBackground),
                 modifier = Modifier.fillMaxWidth()
             )
 
             OutlinedTextField(
                 value = summary,
                 onValueChange = onSummaryChange,
-                label = { Text(stringResource(R.string.info_item_content_summary)) },
+                label = { Text(stringResource(R.string.info_item_content_summary), color = MaterialTheme.colorScheme.onBackground) },
+                textStyle = TextStyle(color = MaterialTheme.colorScheme.onBackground),

```

Comment:
[See this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847123849).

Reply:
This has not been solved.